client/src/components/select/index.js

```diff
@@ -1,16 +1,24 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
-import './select.css';
+import "./select.css";
 
 class Select extends Component {
+  selectValue = e => {
```

Comment:
You might need to add the class properties babel plugin for this class property syntax to work (to get transformed properly in the build step).

It's really easy - you can take a look here:  https://www.npmjs.com/package/babel-plugin-transform-class-properties
It's just an npm package, and then one line added to your .babelrc
We use class properties a lot at my work!  We love them.

Reply:
I was trying to find something to get rid of that annoying red alarm. thanks for pointing this out :)

Reply:
np!